.editorconfig

```diff
@@ -0,0 +1,10 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
yeah I had to branch off that branch because I needed some of the stuff and it's not been merged yet into master.  I did put in the description and mentioned it on slack